PHP-ARRAYS/php-b8.php

```diff
@@ -0,0 +1,47 @@
+<?php 
+
+    $itens = [];
+    $preco = [];    
+    $i = 0;
+    $maiorPreco = 0;
+    $maiorIten = 0;
+
+    function calculoCustos ($valorItens, $part)
```

Comment:
Essa função contém alguns erros. Se o churrasco for cancelado eu não devo imprimir o custo por pessoa, uma vez que não teve o churrasco. Outro ponto, se eu colocar que teve 0 pessoas no churrasco, o código quebra.